tests/unit/TestIntegrator.cpp

```diff
@@ -360,11 +361,11 @@ TEST(IntegratorTest, TestFailAddRead)
                                     true, true)));
 }
 
-TEST(IntegratorTest, TestSucessAddRead)
+TEST(IntegratorTest, TestSuccessAddRead)
 {
     const string tpl = "AA";
     const vector<uint8_t> pw(tpl.length(), 1);
-    auto mdl = P6C4;
+    const auto mdl = P6C4;
```

Comment:
goddammit. I thought that was lumped with the new test I wasn't going to add here. Seems I goofed the interactive add.